conformance_pack/efs.sp

```diff
@@ -59,6 +59,16 @@ control "efs_file_system_protected_by_backup_plan" {
   })
 }
 
+control "efs_file_system_restrict_public_access" {
+  title       = "EFS file systems should restrict public access"
+  description = "Manage access to resources in the AWS Cloud by ensuring EFS file systems cannot be publicly accessed."
+  query       = query.efs_file_system_restrict_public_access
+
+  tags = merge(local.conformance_pack_efs_common_tags, {
+    other_checks = "true"
+  })
+}
+
```

Comment:
@khushboo9024 @Priyanka-Chatterjee-2000 Will remove this one from the list to test more on the file system policy in the next batch to add. Thank you